### PR TITLE
feat(Image): implement `focalPoint` prop

### PR DESCRIPTION
**Pull request by nitayneeman**
### 🔦 Summary

This PR implements `focalPoint` prop supporting a list of presents or a custom point.

From the visuals:
image
image

### CODE REVIEW

File: packages/wix-ui-tpa/src/components/Image/ImageUtils.spec.tsx [hunk 1]
```diff
@@ -1,74 +1,105 @@
...
-    expect(calculatedDimensions).toBeNull();
+      expect(coordinates).toEqual(sampleCoordinates);
+    });
+
+    it('should return `null` when having incorrect preset', async () => {
+      const incorrectPreset = 'Something';
+      const coordinates = resolveFocalPointCoordinates(incorrectPreset);
+
+      expect(coordinates).toBeNull();
+    });
```
Comment (nitayneeman) [inline packages/wix-ui-tpa/src/components/Image/ImageUtils.spec.tsx:103]: This is the relevant part, the rest is only wrapping by `describe`.

File: packages/wix-ui-tpa/src/components/Image/Image.tsx [hunk 2]
```diff
@@ -119,7 +127,14 @@ export class Image extends React.Component<ImageProps> {
             src={src}
             className={classes.image}
             {...(calculatedDimensions && {
-              nativeProps: { ...calculatedDimensions },
+              nativeProps: {
+                ...calculatedDimensions,
+                ...(focalPointCoordinates && {
+                  style: {
+                    objectPosition: `${focalPointCoordinates.x}% ${focalPointCoordinates.y}%`,
```
Comment (nitayneeman) [inline packages/wix-ui-tpa/src/components/Image/Image.tsx:134]: This is the solution for absolute URL

File: packages/wix-ui-tpa/src/components/Image/Image.tsx [hunk 3]
```diff
@@ -130,6 +145,7 @@ export class Image extends React.Component<ImageProps> {
             className={classes.image}
             sourceDimensions={sourceDimensions}
             containerDimensions={calculatedDimensions}
+            focalPoint={focalPointCoordinates}
```
Comment (nitayneeman) [inline packages/wix-ui-tpa/src/components/Image/Image.tsx:148]: This is the solution for relative media URI

File: packages/wix-ui-tpa/src/components/Image/types.ts [hunk 4]
```diff
@@ -41,6 +43,23 @@ export const AspectRatioPresets = {
   landscape: 4 / 3,
 };
 
+export interface FocalPoint {
+  x: number;
+  y: number;
+}
+
+export const FocalPointPresets: { [key: string]: FocalPoint } = {
```
Comment (nitayneeman) [inline packages/wix-ui-tpa/src/components/Image/types.ts:51]: Based on Bookings presets

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6738

**Review by jonathanadler — APPROVED** [review]
Looks great!

**Pull request merged**